sys-fs/xfsprogs/xfsprogs-9999.ebuild

```diff
@@ -0,0 +1,192 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic systemd udev
+
+DESCRIPTION="XFS filesystem utilities"
+HOMEPAGE="https://xfs.wiki.kernel.org/ https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	inherit autotools
+	REGEN_BDEPEND="
+		dev-build/autoconf
+		dev-build/automake
+		dev-build/libtool
+	"
+	EGIT_REPO_URI="https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git"
+	EGIT_BRANCH="for-next"
+	EGIT_CLONE_TYPE="shallow"
```

Comment:
Per `git-r3.eclass` docs, this is a user variable and must not be set by ebuilds.